index.html

```diff
@@ -1 +1,146 @@
+                    <a class="mobile-menu-items sponsor-section" href="#sponsor">Sponsor</a>
+                    <a class="mobile-menu-items news-section" href="#">News</a> 
+                </div>
+            </nav>
 
+        </header>
+        <main>
+            <section class="homePage">
+                <div class="mainContent">
+                    <div class="headingContainer">
+                        <h2 class="greetingHeading">"Hello!Beauty  world"</h2>
+                        <h1 class="titleHeading">SUMMIT OF BEAUTY 2022</h1>
+                    </div>
+                    <div class="mainDescription">
+                        <p>Connect with the best experts, in real life, at our summit.
+                         Be part of this unique experience and listen to their
+                          life-changing advice in person!
+                        </p>
+                    </div>
+                    <div class="date-venue">
+                        <p class="date">2022.10.15(THU) ~ 16(FRI)</p>
+                        <p>@ National Museum of Dubai, Art Center Dubai and more</p>
+                    </div>
+                </div>
+            </section>
+            <section class="programSection" id="Program">
+                <div class="program">
+                    <h2>Main Program</h2>
+                    <div class="line"></div>
+                    <div class="programContainer">
+                        <div class="programItem">
+                            <img src="./Assets/makeup.svg" alt="lecture">
+                            <h3>Make-up</h3>
+                            <p>Handpicked from the most reputable brands to provide you the best</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/skincare.svg" alt="exhibition">
+                            <h3>Skin care</h3>
+                            <p>More than just skin deep, Vegan and cruelty-free</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/haircare.svg" alt="forum">
+                            <h3>Hair care</h3>
+                            <p>Say goodbye to bad chemicals and unhealthy ingredients.</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/workshop.svg" alt="workshop">
+                            <h3>Workshop</h3>
+                            <p>More panels and workshops than last year</p>
+                        </div>
+                        <div class="programItem">
+                            <img src="./Assets/edit-regular.svg" alt="ignite">
+                            <h3>Q & A</h3>
+                            <p>This is your chance to be part of the conversation.</p>
+                            
+                        </div>
```

Comment:
The pages should use the exact same colors, for text, backgrounds, borders, etc, from the original design. All page elements should have similar spacing, borders, shadows, and paddings than the orignal design. I can see some differences between your design and the actual design, the colors, spacing padding etc as shown below. Kindly check and rectify this anomaly. Below are just a few examples


Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-20 12-54-07](https://user-images.githubusercontent.com/65233860/196954907-fbb0aa89-6a1a-4b26-bea4-4bf8d0e4e72f.png)  |  ![Screenshot from 2022-10-20 12-53-57](https://user-images.githubusercontent.com/65233860/196955077-819a6415-bb06-4663-8536-9c2fa229e279.png)

Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-20 13-03-14](https://user-images.githubusercontent.com/65233860/196956325-3c5ad9cd-d756-4d68-b942-92842a25e344.png)  |  ![Screenshot from 2022-10-20 13-03-27](https://user-images.githubusercontent.com/65233860/196956359-f59012f0-f634-4ce6-bbfe-2fc8869286d3.png)



Your design             |  The actual design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-20 13-14-21](https://user-images.githubusercontent.com/65233860/196958535-2ae1200b-e0bf-4ab1-8b61-8d995d2d1e8a.png)  |  ![Screenshot from 2022-10-20 13-14-01](https://user-images.githubusercontent.com/65233860/196958564-88cc7796-fa4d-406c-bfef-2b8b89be9caa.png)